rules/corridor-conversations/rule.md

```diff
@@ -0,0 +1,24 @@
+---
+type: rule
+title: Do you have Corridor Conversations?
+uri: corridor-conversations
+authors:
+  - title: Brady Stroud
+    url: https://ssw.com.au/people/brady-stroud
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+related: []
+redirects: []
+created: 2022-04-21T02:42:50.085Z
+guid: baa5c57c-a2e9-4010-aff8-b468cccd3efc
+---
+
+![Figure: Use corridor conversations to prevent nasty surprises](ProjectManagement\_Suprise.jpg)
+
+The hallway is your friend. It's a place where you can gather a lot of information informally.
+
+Formal meetings can have a "Us vs Them" feel - While the information collected and the conclusions of the Spec Review are presented formally at the end of the Review, it is important that the consultants convey key points to the client as they emerge through the course of the Review. The formal presentation is **not** the time to be presenting new information to the client. 
+
+Addressing key potential sticking points of budget and technology informally during the course of the Spec Review relieves the potential for unwelcome surprises during the Spec Review presentation. Canvassing the issues beforehand in casual 'corridor conversations' clears the decks for an agreement, rather than increasing the risk of heated discussions if you surprise a client at a formal meeting. 
+
+For example, ask the client *"...building the cube will add around 2 months of development time, shall we leave this out of the current scope, or do you want it in?"*
```

Comment:
Is it worth adding a list of example topics for corridor conversations?  For example:
- Estimated Cost and Time
- Key Technologies
- Key Infrastructure
- etc